system/Config/BaseService.php

```diff
@@ -300,6 +300,17 @@ public static function reset(bool $initAutoloader = false)
 		}
 	}
 
+	/**
+	 * Resets any mock and shared instances for a single service.
+	 *
+	 * @param string $name
+	 */
+	public static function resetSingle(string $name)
+	{
+		unset(static::$mocks[$name]);
+		unset(static::$instances[$name]);
```

Comment:
Combine as one call to `unset`.

Can we also check if the name is set first before unsetting.

Reply:
Will combine!

> Can we also check if the name is set first before unsetting.

What is the benefit to this? `unset` does not care so it adds unnecessary steps.

Reply:
Hmm. Ok. I was today years old when I learned unsetting an inexistent variable raises no error. 🤔🤪